### DISCUSSION TITLE
Test the glove w/o the VR headset

### DISCUSSION THREAD

**Discussion by LeBandicoot** [LucidVR/opengloves-driver#67] [Help & Troubleshooting] [answered]
Do I necessarily need the VR headset to render my palm and the finger tracking on the computer?

**Comment by danwillm** [comment, marked as answer]
You do, yes. This driver works with SteamVR which requires a headset for use typically.

There's a couple of other projects that allow you to use an emulated headset, and move the headset and controllers with keybinds, this one may help: https://github.com/r57zone/OpenVR-driver-for-DIY

**Comment by PerlinWarp** [comment]
Hey @LeBandicoot , If it's useful, I wrote some python code to render the fingers without VR.

![Demo showing hand and finger movement](https://media2.giphy.com/media/4P8L3HQAg5qEDBmffP/giphy.gif?cid=790b7611cd5a3029ab0414bbdc674fb1a19867a290c3426a&rid=giphy.gif&ct=g)

Currently I've implemented it over Serial, testing with the Arduino Nano in Legacy and Alpha encodings.
The code is here:  [pygloves-utils](https://github.com/PerlinWarp/pygloves-utils)
